db/changes.go

```diff
@@ -132,11 +132,11 @@ func (db *Database) addDocToChangeEntry(entry *ChangeEntry, options ChangesOptio
 }
 
 func (db *Database) AddDocToChangeEntryUsingRevCache(entry *ChangeEntry, revID string) (err error) {
-	rev, err := db.getRev(entry.ID, revID, 0, nil)
+	rev, err := db.getRev(entry.ID, revID, 0, nil, RevCacheIncludeBody)
 	if err != nil {
 		return err
 	}
-	entry.Doc, err = rev.As1xBytes(db, nil, nil, false)
+	entry.Doc, err = rev.As1xBytes(db, nil, nil, RevCacheOmitBody)
```

Comment:
This parameter is only related to expiry, don't think it should be using the omit body constant.

```suggestion
	entry.Doc, err = rev.As1xBytes(db, nil, nil, false)
```

Reply:
Agreed - will fix